smurfs/src/components/SmurfCard.js

```diff
@@ -0,0 +1,14 @@
+import React from 'react';
```

Comment:
add some comments on all of your files, so that you can tell whats going on if you come back to it at a later date